src/Native/System.Native/pal_runtimeinformation.h

```diff
@@ -8,11 +8,14 @@
 
 extern "C" int32_t SystemNative_GetUnixVersion(char* version, int* capacity);
 
-extern "C" int32_t SystemNative_GetUnixArchitecture();
+extern "C" int32_t SystemNative_GetOSArchitecture();
```

Comment:
My guess is that it'll be ok as-is in this case, but the CI system should tell us.  The problem requires a specific set of circumstances to manifest, and I don't think those conditions are met here, but we'll see.